css-cascade-6/Overview.bs

```diff
@@ -351,17 +351,49 @@ Scoping Styles: the ''@scope'' rule</h4>
 
 	The syntax of the ''@scope'' rule is:
 
-	<pre class='prod'>
-	@scope (<<scope-start>>) [to (<<scope-end>>)]? {
+	<pre class="prod def">
+	@scope [ (<<scope-start>>) [to (<<scope-end>>)]? ]? {
 	  <<stylesheet>>
 	}
 	</pre>
 
+	where:
+
+	<pre class="prod def">
+	<dfn><<scope-start>></dfn> = <<forgiving-selector-list>>
+	<dfn><<scope-end>></dfn> = <<forgiving-selector-list>>
+	</pre>
+
+	A ''@scope'' rule which specifies a <<scope-start>>
+	produces a number of <dfn>explicit scopes</dfn> as follows:
+
+	* For each element matched by <<scope-start>>,
+		create a [=scope=] using that element as the [=scoping root=].
+	* For each [=scope=] created by the above:
+		* Collect all elements that are [=in scope=]
+			and that match <<scope-end>>,
+			using the [=scoping root=] as the '':scope'' element, then
+		* Set those elements as the [=scoping limits=].
+
+	[=Pseudo-elements=] cannot be [=scoping roots=] or [=scoping limits=];
```

Comment:
After moving this above in the previous PR, I moved it back down! 😅

Reply:
:-)